### PR TITLE
Clarify technology choices

### PR DESCRIPTION
**Pull request by arturohernandez10**
## Summary

Describe your changes here.

Since  *.NET Core/5+* is the latest and greatest. Let's not muddle the waters, by having an unspecified *.NET Framework* above. If there is something else Microsoft would like to let us know, please be clear.

### CODE REVIEW

File: docs/standard/choosing-core-framework-server.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ ms.date: 04/20/2021
 
 There are two supported .NET implementations for building server-side apps:
 
-* *.NET Framework*
 * *.NET Core/5+*, which includes .NET Core, .NET 5, and later versions.
+* *.NET Framework*, legacy versions.
```
Comment (tdykstra) [inline docs/standard/choosing-core-framework-server.md:12]: Reversing the order and clarifying versions here makes sense, but "legacy" has connotations that don't apply to .NET Framework.

```suggestion
* *.NET Core/5+*, which includes .NET Core, .NET 5, and later versions.
* *.NET Framework*, which includes 4.8 and earlier versions.
```

**Review by tdykstra — CHANGES_REQUESTED** [review]
Thanks for the submission! I have one suggestion that might help make the text clearer.

**Comment by tdykstra** [comment]
@gewarren Your thoughts on the proposed change?

**Comment by gewarren** [comment]
> @gewarren Your thoughts on the proposed change?

I like your proposal better.

**Review by tdykstra — APPROVED** [review]
This is a good idea. It might make it clear enough that you could consider changing ".NET Core/5+" to ".NET" throughout.

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25684) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25684) before we can accept your contribution.<br/><br/>:x: arturohernandez10 [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25684)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=25684) it.</sub>

**Comment by gewarren** [comment]
@tdykstra I applied your suggestion in the latest commit. Can you look and see what you think?

File: docs/standard/choosing-core-framework-server.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+title: Choose between .NET and .NET Framework for server apps
 description: A guide to help you decide which implementation of .NET to use when building a server app.
 author: cartermp
 ms.date: 04/20/2021
 ---
-# .NET Core/5+ vs. .NET Framework for server apps
+# .NET vs. .NET Framework for server apps
 
-There are two supported .NET implementations for building server-side apps:
+There are two supported .NET implementations for building server-side apps.
```
Comment (tdykstra) [inline docs/standard/choosing-core-framework-server.md:9]:
```suggestion
There are two supported [.NET implementations](glossary.md#implementation-of-net) for building server-side apps.
```

**Review by tdykstra — APPROVED** [review]
I added one suggestion. The multiple meanings of ".NET" could still be confusing but that's inherent in the officially approved terminology, and this limits the multiple meanings to the intro where they are explained. I'm inclined to go ahead with this approach.

**Review by gewarren — APPROVED** [review]

**Comment by gewarren** [comment]
@arturohernandez10 Could you sign the license agreement, and then we can merge this?

**Comment by arturohernandez10** [comment]
@gewarren I just signed. Thanks!

**Pull request merged**